generators/create-react-app/templates/base/package.json.ejs

```diff
@@ -2,6 +2,7 @@
   "name": "<%= packageName %>",
   "version": "0.1.0",
   "private": true,
+  "homepage": "http://example.com<%= httpPath %>",
```

Comment:
`homepage` is supposed to be a valid URL so the domain is needed but since CRA ignores the domain part I chose to use example.com.

But it looks like it works with a relative URL so :+1: